DNN Platform/Modules/DnnExportImport/Components/Common/Util.cs

```diff
@@ -2,284 +2,381 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
 
-namespace Dnn.ExportImport.Components.Common
-{
-    using System;
-    using System.Collections.Generic;
-    using System.IO;
-    using System.Reflection;
-    using System.Text;
-    using System.Threading;
-
-    using Dnn.ExportImport.Components.Controllers;
-    using Dnn.ExportImport.Components.Entities;
-    using Dnn.ExportImport.Components.Providers;
-    using Dnn.ExportImport.Components.Services;
-    using DotNetNuke.Common;
-    using DotNetNuke.Common.Utilities;
-    using DotNetNuke.Entities.Modules.Definitions;
-    using DotNetNuke.Entities.Profile;
-    using DotNetNuke.Entities.Users;
-    using DotNetNuke.Framework.Reflections;
-    using DotNetNuke.Instrumentation;
-    using Newtonsoft.Json;
-
-    public static class Util
-    {
-        private static readonly ILog Logger = LoggerSource.Instance.GetLogger(typeof(Util));
-        private static int _noRole = Convert.ToInt32(Globals.glbRoleNothing);
-
-        // some string extension helpers
-        public static bool IsNullOrEmpty(this string s) => string.IsNullOrEmpty(s);
-
-        public static bool IsNullOrWhiteSpace(this string s) => string.IsNullOrWhiteSpace(s);
-
-        public static bool HasValue(this string s) => !string.IsNullOrEmpty(s);
```

Comment:
We [ship these functions](https://www.fuget.org/packages/DotNetNuke.SiteExportImport/9.7.1/lib/net45/DotNetNuke.SiteExportImport.dll/Dnn.ExportImport.Components.Common/Util?code=true#M%3ADnn.ExportImport.Components.Common.Util.IsNullOrEmpty%28System.String%29) (this project is [published as a NuGet package](https://www.nuget.org/packages/DotNetNuke.SiteExportImport/9.7.1)), so I don't think we can remove them without deprecating.

Reply:
Oh, makes sense, I had no clue we published that on Nuget. I have restored them and deprecated them